avocado/utils/distro.py

```diff
@@ -343,7 +395,7 @@ class SUSEProbe(Probe):
     CHECK_FILE_DISTRO_NAME = 'SuSE'
 
     def get_distro(self):
-        distro = Probe.get_distro(self)
+        distro = super().get_distro(self.session)
```

Comment:
Since `get_distro()` doesn't have session as argument, this first argument is about self which is already passed:

```python
distro = super().get_distro()
```